tests/test_project_status.py

```diff
@@ -365,6 +367,39 @@ def test_set_project_to_available_valid_transition(module_client, test_project):
     assert db_deadline == calc_deadline
 
 
+def test_set_project_to_available_no_mail(client):
```

Comment:
```suggestion
def test_set_project_to_available_no_mail(client, test_project):
```

Reply:
I think it should be boto3_session as that is what we use in the project creation tests.
```suggestion
def test_set_project_to_available_no_mail(client, boto3_session):
```

Reply:
Thanks a lot! I now went with `def test_set_project_to_available_no_mail(client, boto3_session)`